docs/sources/setup/_index.md

```diff
@@ -8,5 +8,6 @@ weight: 300
 # Setup Loki
 
 - [Install](install/) Loki.
+  - [Grafana Log Dashboard Guide](install/grafana-log-dashboard-guide/)
```

Comment:
This link shouldn't be added to this page, as we're explaining the categories here, not linking to individual topics.